go.sum

```diff
@@ -1,14 +1,12 @@
-github.com/99designs/gqlgen v0.17.20 h1:O7WzccIhKB1dm+7g6dhQcULINftfiLSBg2l/mwbpJMw=
-github.com/99designs/gqlgen v0.17.20/go.mod h1:Mja2HI23kWT1VRH09hvWshFgOzKswpO20o4ScpJIES4=
```

Comment:
[dependency] The version of github.com/hashicorp/golang-lru has been updated from v0.5.4 to v2.0.1. It is recommended to check if this update introduces any breaking changes or new features that may affect the software.